daemon/create_unix.go

```diff
@@ -60,14 +61,24 @@ func (daemon *Daemon) createContainerPlatformSpecificSettings(container *contain
 			return err
 		}
 
-		// never attempt to copy existing content in a container FS to a shared volume
-		if v.DriverName() == volume.DefaultDriverName {
```

Comment:
We talked about it on the review call and said it should happen for any driver.
I believe this part was to make sure this wasn't a bind? I'll have to pull up that PR again.